crates/hc_demo_cli/src/test.rs

```diff
@@ -13,6 +13,78 @@ const CONTENT: &[u8] = b"this is a test\n";
 #[tokio::test(flavor = "multi_thread")]
 #[cfg_attr(target_os = "macos", ignore = "flaky")]
 async fn demo() {
+    run_test(|| async {
+        let (r1s, r1r) = tokio::sync::oneshot::channel();
+        let (r2s, r2r) = tokio::sync::oneshot::channel();
+        let rendezvous = holochain::sweettest::SweetLocalRendezvous::new().await;
+
+        let t1 = tokio::task::spawn(run("one", r1s, rendezvous.clone()));
+        let t2 = tokio::task::spawn(run("two", r2s, rendezvous.clone()));
+
+        let _ = r1r.await;
+        let _ = r2r.await;
+
+        (t1, t2)
+    })
+    .await;
+}
+
+#[tokio::test(flavor = "multi_thread")]
+#[cfg_attr(target_os = "macos", ignore = "flaky")]
```

Comment:
I suggest to at least create an issue to fix these flaky tests. Otherwise we just "ensure holochain runs with multiple signal server usage" on Linux ;-)

Reply:
What is our policy on this? I don't believe we write issues for every test we make flaky.

Reply:
We don't, I agree. To me the todo is that none of the tests pass under macos.

Reply:
Gottcha. I'm pretty sure the tests run fine when run individually on macOs. They just flake when run as part of CI.